packages/components/src/InputPhoneNumber/InputPhoneNumber.tsx

```diff
@@ -14,7 +14,8 @@ interface InputPhoneNumberProps
       | "readonly"
       | "prefix"
       | "suffix"
-    > {
+    >,
+    Partial<Pick<InputMaskProps, "pattern">> {
```

Comment:
Might be best if this is actually written as 

```ts
{
  ...
  /*
   * Custom description
   */
  readonly pattern: InputMaskProps["pattern"]
}
```

since the input mask description doesn't really apply here 
![image](https://user-images.githubusercontent.com/15986172/236552088-afb481b0-3f81-4280-a623-37298cb9a0f5.png)